src/test/java/com/clouway/cuse/SearchEngineContractTest.java

```diff
@@ -993,6 +1027,15 @@ private void store(User... users) {
     }
   }
 
+  private void storeAllUsers(List<User> users) {
```

Comment:
I don't see this used anywhere.
Make a happy path test with that, I think it will be enough.